Code/GraphMol/Descriptors/MolSurf.cpp

```diff
@@ -357,5 +361,20 @@ std::vector<double> calcPEOE_VSA(const ROMol &mol, std::vector<double> *bins,
 
   return res;
 }
+
+std::vector<double> calcCustomProp_VSA(const ROMol &mol, const std::string &customPropName,
+		const std::vector<double> &bins, bool force) {
+  std::vector<double> res(bins.size() + 1, 0);
+
+  std::vector<double> vsaContribs(mol.getNumAtoms());
+  double tmp;
+  getLabuteAtomContribs(mol, vsaContribs, tmp, true, force);
+
+  std::vector<double> prop(mol.getNumAtoms(), 0.0);
+  prop = moldata3D.GetCustomAtomProp(mol, customPropName);
```

Comment:
You could just as easily fill `prop` by looping over the atoms and retrieving the property values directly.